frontend/src/constants/API.tsx

```diff
@@ -33,7 +33,7 @@ export const PARCELS = (params: IParcelListParams | null) =>
 export interface IParcelDetailParams {
   id: number;
 }
-export const PARCEL_DETAIL = (params: IParcelDetailParams) => `/parcels/${params.id}`;
+export const PARCEL_DETAIL = (params: IParcelDetailParams) => `/properties/${params.id}`;
```

Comment:
An endpoint does not exist for `/properties/{id}`.  You'll need to make a request to the correct endpoint based on the `PropertyTypeId`.

Parcel = `/parcels/{id}`
Building = `/buildings/{id}`

`/properties` returns an array of *property* objects.
```
{
  propertyTypeId: 0 or 1
  ...
}
```
Parcel = 0
Building = 1

Reply:
yeah I haven't made the changes yet. I just rebased and force pushed my branch. I will remove the **DRAFT PR** label when this is ready for review and ask the team to review. Until then this is in draft mode